python/artexplainer.Dockerfile

```diff
@@ -9,7 +9,8 @@ ARG POETRY_HOME=/opt/poetry
 ARG POETRY_VERSION=1.7.1
 
 # Required for building packages for arm64 arch
-RUN apt-get update && apt-get install -y --no-install-recommends python3-dev build-essential
+RUN apt-get update && apt-get install -y --no-install-recommends python3-dev build-essential pkg-config libhdf5-dev && \
```

Comment:
for arm we could pin the h5py version to 2.10 (IIRC) instead doing it here.
wdyt?

Reply:
We already doing some compiling for ARM64 and after reading the [comment here](https://github.com/h5py/h5py/pull/2405#issuecomment-2047248192) I am not sure when the ARM builds will be available.

Reply:
Oh, for now we could stick with 3.10 and wait for that. =/

Reply:
we still can use h5py with newer python versions, right?